src/text/line.rs

```diff
@@ -474,12 +474,24 @@ impl<'a> Line<'a> {
             let new_span_width = span_width - offset;
             if x + new_span_width > result_width {
                 let span_end = (result_width - x + offset) as usize;
-                new_span.content = Cow::from(&span.content[offset as usize..span_end]);
+                new_span.content = Cow::from(
+                    span.content
+                        .as_ref()
```

Comment:
`.chars()` works on the content directly. Therefore, it's shorter to remove the `.as_ref()` call as that happens automatically.
```suggestion
```